.changelogs/order-information.yml

```diff
@@ -0,0 +1,5 @@
+significance: minor
+type: changed
+links:
+  - "#2033"
+entry: The coupon code in the student's order details table is now wrapped in a `code` instead of a `a` tag.
```

Comment:
```suggestion
entry: The coupon code in the student's order details table is now wrapped in a `<code>` tag instead of an `<a>` tag.
```

Reply:
you now that I first wrote `an a` and then I remembered our conversation and ended up saying "jeez I don't know, let's try with just a" :D